src/openzaak/management/commands/setup_configuration.py

```diff
@@ -0,0 +1,104 @@
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from vng_api_common.authorizations.models import Applicatie, Autorisatie
+from vng_api_common.constants import ComponentTypes
+from vng_api_common.models import APICredential, JWTSecret
+from vng_api_common.notifications.constants import (
+    SCOPE_NOTIFICATIES_CONSUMEREN_LABEL,
+    SCOPE_NOTIFICATIES_PUBLICEREN_LABEL,
+)
+from vng_api_common.notifications.models import NotificationsConfig
+
+from openzaak.components.autorisaties.api.scopes import SCOPE_AUTORISATIES_LEZEN
+
+
+class Command(BaseCommand):
+    help = "Setup the initial necessary configuration"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "openzaak_domain",
+            help="Specifies the domain for this Open Zaak installation",
+        )
+        parser.add_argument(
+            "notifications_api_root",
+            help="Specifies the API root for the notifications API.",
+        )
+        parser.add_argument(
+            "municipality",
+            help="Name of the municipality to which the Open Zaak installation belongs",
+        )
+        parser.add_argument(
+            "openzaak_to_notif_secret",
+            help="Secret used for the Application that allows Open Zaak to retrieve notifications",
+        )
+        parser.add_argument(
+            "notif_to_openzaak_secret",
+            help="Secret used for the Application that allows Notifications API to retrieve authorizations",
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        try:
+            Applicatie.objects.all().delete()
+            openzaak_domain = options["openzaak_domain"]
+            notifications_api_root = options["notifications_api_root"]
+            municipality = options["municipality"]
+            openzaak_to_notif_secret = options["openzaak_to_notif_secret"]
+            notif_to_openzaak_secret = options["notif_to_openzaak_secret"]
+
+            site = Site.objects.get_current()
```

Comment:
Please add references to the config docs to all sections.

```suggestion
            # See: https://open-zaak.readthedocs.io/en/latest/installation/configuration.html#setting-the-domain
            site = Site.objects.get_current()
```